### ISSUE TITLE
OnePlus support

### ISSUE THREAD

**Issue by devsnek** [WireGuard/android-wireguard-module-builder#2]
It would be great to see support for OnePlus devices. I'd also be happy to test builds for the OnePlus 5.

Kernel sources:

- https://github.com/OnePlusOSS/android_kernel_oneplus_msm8998/tree/oneplus/QC8998_P_9.0
- https://github.com/OnePlusOSS/android_kernel_oneplus_msm8996/tree/oneplus/QC8996_P_9.0

**Comment by nathanchance** [comment]
As I have told Jason, my time is limited nowadays more than ever. I will not be able to do this for a very long time if ever. I would be more than happy to review someone else's work and help get it merged though.

**Comment by devsnek** [comment]
I'd be happy to help but I'm facing the same problem as #1

**Comment by nathanchance** [comment]
I will try to document that eventually (again, time) but the other issue with OnePlus is that they will need build.config files, which is a much harder thing to document.

**Comment by zx2c4** [comment]
@nathanchance Perhaps you can find somebody else to help you with the mantainance burden instead of abandoning this? You know lots of Android people, I imagine.

**Comment by nathanchance** [comment]
@zx2c4 Yes, that would be ideal. I'll have to take a look at the OnePlus scene and see who is still around. The main issue is most kernel developers are running their own stuff rather than stock so they do not have much incentive to working on something like this.
